iset.mm

```diff
@@ -123738,6 +123738,249 @@ seq m ( x. , F ) ~~> x ) ) \/
+       (Contributed by Scott Fenton, 11-Dec-2017.)  (Revised by Jim Kingdon,
+       6-Aug-2024.) $)
+    prodssdc $p |- ( ph -> prod_ k e. A C = prod_ k e. B C ) $=
+      ( c1 cc wceq vm cv cmpt cfv cprod cmul cuz wcel cif cseq cli sstrd wa csb
+      eqid simpr wn wo wdc eleq1w wral adantr rspcdva exmiddc syl iftrue adantl
+      dcbid wi cdif eldif ax-1cn eqeltrdi sylan2br expr sselda mpjaod ralrimiva
+      nfcsb1v nfel1 csbeq1a eleq1d rspc mpan9 eqeltrd iffalse a1i jaod mpd nfcv
+      nfv nfif ifbieq1d fvmptf syl2anc syldan fvmpts eqtrd eqeq1d eqtr4d ssneld
+      ex nfeq1 imp fmpttd ffvelrnda zproddc ifeq1d prodfct 3eqtr3d ) ACUAUBZGCE
+      UCZUDZUAUEZDXKGDEUCZUDZUAUEZCEGUEZDEGUEZAXNUFGIUGUDZGUBZDUHZERUIZUCZIUJUK
+      UDXQABCXMFUAHYDIXTXTUOZNLACDXTJPULMAXKXTUHZUMZXKYDUDZXKDUHZGXKEUNZRUIZXKC
+      UHZXMRUIZYGYFYKSUHZYHYKTAYFUPZYGYIYIUQZURZYNYGYIUSZYQYGFUBZDUHZUSZYRFXTXK
+      YSXKTZYTYIFUADUTVHAUUAFXTVAYFQVBYOVCYIVDVEZAYQYNVIYFAYIYNYPAYIYNAYIUMZYKY
+      JSYIYKYJTAYIYJRVFVGZAESUHZGDVAZYIYJSUHZAUUFGDAYBUMZYACUHZUUFUUJUQZAUUJUUF
+      VIYBAUUJUUFKXBVBAYBUUKUUFYBUUKUMAYADCVJZUHZUUFYADCVKAUUMUMERSOVLVMVNVOUUI
+      UUJUSZUUJUUKURUUIYSCUHZUSZUUNFXTYAYSYATUUOUUJFGCUTVHAUUPFXTVAZYBMVBADXTYA
+      PVPVCUUJVDVEVQZVRZUUFUUHGXKDGYJSGXKEVSZVTYAXKTZEYJSGXKEWAZWBWCWDZWEXBYPYN
+      VIAYPYKRSYIYJRWFZVLVMWGWHVBWIGXKYCYKXTYDSGXKWJYIGYJRYIGWKUUTGRWJWLUVAYBYI
+      EYJRGUADUTUVBWMYDUOWNWOZYGYQYMYKTZUUCAYQUVFVIYFAYIUVFYPAYIUVFUUDYMYJYKUUD
+      YLYMYJTZYLUQZAYLUVGVIYIAYLUVGAYLUMZYMXMYJYLYMXMTAYLXMRVFVGUVIYLUUHXMYJTAY
+      LUPAYLYIUUHACDXKJVPUVCWPGXKECXLSXLUOWQWOWRXBVBAYIUVHUVGAYIUVHUMZUMYMRYJUV
+      JYMRTZAUVHUVKYIYLXMRWFZVGVGUVJAXKUULUHZYJRTZXKDCVKAERTZGUULVAUVMUVNAUVOGU
+      ULOVRUVOUVNGXKUULGYJRUUTXCUVAEYJRUVBWSWCWDVNWTVOUUDYLUSZYLUVHURUUDUUPUVPF
+      XTXKUUBUUOYLFUACUTVHAUUQYIMVBADXTXKPVPVCYLVDVEVQUUEWTXBAYPUVFAYPUMZYMRYKU
+      VQUVHUVKAYPUVHACDXKJXAXDUVLVEYPYKRTAUVDVGWTXBWHVBWIWTACSXKXLAGCESKXEXFXGA
+      BDXPFUAHYDIXTYENLPQYGYHYKYIXPRUIZUVEYGYQUVRYKTZUUCAYQUVSVIYFAYIUVSYPAYIUV
+      SUUDYIXPYJRUUDYIUUHXPYJTAYIUPUVCGXKEDXOSXOUOWQWOXHXBYPUVSVIAYPUVRRYKYIXPR
+      WFUVDWTWGWHVBWIWTADSXKXOAGDESUURXEXFXGWTAUUFGCVAXNXRTAUUFGCKVRCEUAGXIVEAU
+      UGXQXSTUUSDEUAGXIVEXJ $.
+  $}
+
+  ${
+    $d A j $.  $d A f p $.  $d A k $.  $d A m $.  $d A n $.  $d B f $.
+    $d B k $.  $d B m $.  $d B n $.  $d B j $.  $d C f $.  $d C m p $.
+    $d C n $.  $d f k $.  $d f m $.  $d f n $.  $d f ph $.  $d k m $.
+    $d k n $.  $d k ph $.  $d m n $.  $d m ph $.  $d n ph $.  $d A y $.
+    $d B y p $.  $d C y $.  $d f y $.  $d k p y $.  $d m y $.  $d n y p $.
+    $d ph y p $.  $d j p $.  $d f j $.  $d j k $.
+    fprodss.1 $e |- ( ph -> A C_ B ) $.
+    fprodss.2 $e |- ( ( ph /\ k e. A ) -> C e. CC ) $.
+    fprodssdc.a $e |- ( ph -> A. j e. B DECID j e. A ) $.
+    fprodss.3 $e |- ( ( ph /\ k e. ( B \ A ) ) -> C = 1 ) $.
+    fprodss.4 $e |- ( ph -> B e. Fin ) $.
+    $( Change the index set to a subset in a finite sum.  (Contributed by Scott
```

Comment:
```suggestion
    $( Change the index set to a subset in a finite product.  (Contributed by Scott
```

Reply:
Actually, I'll fix it in iset.mm and set.mm in my upcoming PR.

Reply:
Sounds good. I hadn't even seen that, good eyes.